### DISCUSSION TITLE
Extension launcher

### DISCUSSION THREAD

**Discussion by Sunnyl6** [3kh0/ext-remover#527] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
How do i use the bookmarklet  Launcher?

**Comment by Aka-but-nice** [comment, marked as answer]
You go to <a href="https://chrome.google.com/webstorex">here</a> then use the bookmarklet, you would put the icon of the extension you want, the id, and the name which can be anything.

**Comment by FortniteStephenCurry** [comment]
I tried this, it doesn't work

**Reply by TheGooferFox** [reply]
Mr fortnite does javascript work for you?

**Reply by TheGooferFox** [reply]
What doesn't work? if webstorex is blocked that means u cant do it common sense.....

**Reply by xHydra1337** [reply]
You dont need webstorex, any google error page will work just fine, it really doesnt matter. Try: [chrome.google.com/xxxx](url)

**Comment by FortniteStephenCurry** [comment]
Webstorex isn't blocked, I put in the app id for Hoxx vpn and the name and a letter for the icon and nothing happened.

**Reply by obkduishinijf** [reply]
make sure you don't got a block list, also, this is for EXTENSIONS not apps. 99% sure it won't work for apps, extensions only
